.github/workflows/build-images-push-ecr.yml

```diff
@@ -0,0 +1,44 @@
+name: BetterUp - Build Docker Images and push to ECR
+on:
+  push:
+
+
+jobs:
+  build-gradle-project:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout project sources
+        uses: actions/checkout@v3
+      - uses: actions/setup-java@v3
+        with:
+          distribution: "zulu"
+          java-version: "17"
+      - uses: actions/setup-python@v4
+        with:
+          python-version: "3.9"
+      - uses: actions/setup-node@v3
+        with:
+          node-version: "lts/*"
+      - name: Setup Gradle
+        uses: gradle/gradle-build-action@v2
+      - name: Run build with Gradle Wrapper
+        run: SUB_BUILD=PLATFORM ./gradlew buildDockerImage
+
+      - name: Configure AWS User credentials
+        uses: aws-actions/configure-aws-credentials@v2
+        with:
+          aws-access-key-id: ${{ secrets.GOV_DEV_AWS_ACCESS_KEY_ID }}
```

Comment:
I thought we were pushing to commercial ecr?  I don't think we want to push directly to gov ecr